imp_services/displayLogs.js

```diff
@@ -144,14 +161,18 @@ module.exports =
                         if (LogTypeMap[LogType] == null) {
                             stringLogs.push(typeNotAddedYet(LogType, logUsername, time, JSON.parse(actionData)));
                         } else {
-                            stringLogs.push(LogTypeMap[LogType].callFunction(LogType, logUsername, time, JSON.parse(actionData)));
+                                for (var j = 0; j < filters.length; j++) {
+                                    if (filters[j] == LogType) {
+                                        stringLogs.push(LogTypeMap[LogType].callFunction(LogType, logUsername, time, JSON.parse(actionData)));
+                                    }
+                                }
+
                         }
                         ids.push(logID);
                         //console.log(stringLogs[i]);
                     }
 
-                    var jsonObject = {logs:stringLogs, id:ids};
-
+                    var jsonObject = {"logs":stringLogs, "id":ids};
                     callback(JSON.stringify(jsonObject));
```

Comment:
What is this doing?  If this is logging stuff that doesn't directly pertain to the changes to the location selection that are on the jira ticket it absolutely needs to be pulled into its own branch.